packages/ui/src/views/repo/repo-tags/components/repo-tags-list.tsx

```diff
@@ -1,76 +1,153 @@
-import { CommitCopyActions, MoreActionsTooltip, SkeletonTable, Table } from '@/components'
-import { formatDate } from '@utils/utils'
-import { TranslationStore } from '@views/repo/repo-list/types'
+import { FC, useMemo } from 'react'
 
-import { RepoTagsStore } from '../types'
+import { Avatar, CommitCopyActions, MoreActionsTooltip, NoData, SkeletonTable, Table, Text } from '@/components'
+import { getInitials, timeAgo } from '@/utils'
+import { BranchSelectorListItem, CommitTagType, RepoTagsStore, TranslationStore } from '@/views'
 
 interface RepoTagsListProps {
   useTranslationStore: () => TranslationStore
-  isLoading: boolean
   onDeleteTag: (tagName: string) => void
   useRepoTagsStore: () => RepoTagsStore
   toCommitDetails?: ({ sha }: { sha: string }) => string
-  openCreateBranchDialog: () => void
+  openCreateBranchDialog: (selectedTagInList: BranchSelectorListItem) => void
+  isLoading: boolean
+  isDirtyList: boolean
+  handleResetFiltersAndPages: () => void
+  openCreateTagDialog: () => void
```

Comment:
There's a big mix of naming conventions at use in this component for event handling with `onDeleteTag`, `handleResetFiltersAndPages` and really also `openCreateTagDialog` and `openCreateBranchDialog`. Ideally all things of this type should be `onWhatEverTriggersIt`. The `openX` ones also fall into this category since they are triggered when certain things happen; whether they open a dialog or a take a completely different action should be up to the caller.